SWSabres/Model/Schedule.swift

```diff
@@ -76,32 +76,18 @@ struct Schedule: ResponseJSONObjectSerializable, UniqueObject
             return nil
         }
         
-        let dateFormatter: NSDateFormatter = NSDateFormatter()
-        dateFormatter.locale = NSLocale(localeIdentifier: "en_US_POSIX")
+        let dateFormatter: DateFormatter = DateFormatter()
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
         dateFormatter.dateFormat = "yyyy'-'MM'-'dd HH:mm:ss"
         
-        guard let parsedModified: NSDate = dateFormatter.dateFromString(schedule_modified) else
+        guard let parsedModified: Date = dateFormatter.date(from: schedule_modified) else
         {
             return nil
         }
         
         self.modified = parsedModified
         
-        let encodedTitle = schedule_title.dataUsingEncoding(NSUTF8StringEncoding)!
-        let attributedOptions : [String: AnyObject] = [
-            NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType,
-            NSCharacterEncodingDocumentAttribute: NSUTF8StringEncoding
-        ]
-        
-        if let attributedString = try? NSAttributedString(data: encodedTitle, options: attributedOptions, documentAttributes: nil)
-        {
-            self.title = attributedString.string
-        }
-        else
-        {
-            return nil
-        }
-        
```

Comment:
Scheduled title can have html in it so this parses the html.  Same as venue title this is needed.

Reply:
added this back